scripts/c1.yaml

```diff
@@ -9,5 +9,4 @@ nodes:
     - |
       kind: ClusterConfiguration
       dns:
-        imageRepository: multicluster
-        imageTag: latest
+        imageTag: v1.12.2
```

Comment:
Could you add a small todo or fixme comment to remove this after Kubernetes 1.35 (see https://github.com/kubernetes/kubernetes/pull/132288)?